Configuration/Skimming/src/DisappearingMuonsSkimming.cc

```diff
@@ -17,38 +17,37 @@
 //
 // system include files
 #include <memory>
-
 #include "Math/VectorUtil.h"
+
 // user include files
 #include "Configuration/Skimming/interface/DisappearingMuonsSkimming.h"
-#include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/one/EDFilter.h"
-#include "FWCore/Framework/interface/Event.h"
-#include "FWCore/Framework/interface/MakerMacros.h"
-#include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "DataFormats/Common/interface/TriggerResults.h"
```

Comment:
```suggestion
```
already included in the header

Reply:
yes done in https://github.com/cms-sw/cmssw/commit/7a3a15df8d7e392f2e1394ca2120e5c5f2e6512c